pkg/services/navtree/navtreeimpl/applinks.go

```diff
@@ -359,7 +359,15 @@ func (s *ServiceImpl) addPluginToSection(c *contextmodel.ReqContext, treeRoot *n
 
 func (s *ServiceImpl) hasAccessToInclude(c *contextmodel.ReqContext, pluginID string) func(include *plugins.Includes) bool {
 	hasAccess := ac.HasAccess(s.accessControl, c)
+	ctx := c.Req.Context()
 	return func(include *plugins.Includes) bool {
+		if include.ReqFeatureToggle != "" && !s.features.IsEnabled(ctx, include.ReqFeatureToggle) {
```

Comment:
Is this the same as `omitEmpty`, will this still be an empty string, or will it not exist??